src/unitxt/metrics.py

```diff
@@ -1417,6 +1418,99 @@ def compute(
         return self.pipe(inputs, batch_size=self.batch_size)
 
 
+class LlamaIndexCorrectnessMetric(BulkInstanceMetric):
```

Comment:
You should add a test case to the code.  You may ask, how to test without actually calling the OpenAI API. 
This can be done with Mockup (see test_load_from_ibm_cos  in test_loaders) - but it's a bit tricky if you have not done so before (I did it there, and it's not 100% clear how to do it here).

You would need to mock these two calls:

llm = OpenAI("gpt-3.5-turbo")
self.evaluator = CorrectnessEvaluator(llm=llm)